blocks/form/form.css

```diff
@@ -65,21 +65,35 @@ main .form .form-checkbox-wrapper {
 }
 
 main .form .form-checkbox-wrapper input[type='checkbox'] {
-  display: none;
-}
+  appearance: none;
+
+  /* stylelint-disable */
+  -webkit-appearance: none;
+  -moz-appearance: none;
```

Comment:
```suggestion
```

https://caniuse.com/css-appearance says you don't need the vendor prefixes anymore.

Reply:
I also thought and hence didn't add that in the original commit. But when I tested that in the safari on my browser, i found it not to be working and hence updated it (please see the screenshots)